zookeeper-server/src/main/java/org/apache/zookeeper/server/DataTree.java

```diff
@@ -1001,6 +1001,7 @@ public ProcessTxnResult processTxn(TxnHeader header, Record txn, boolean isSubTx
                     Record record = null;
                     switch (subtxn.getType()) {
                     case OpCode.create:
+                    case OpCode.create2:
```

Comment:
this change is interesting here, because it means that this feature will be supported only when the server is 3.7+

which error will be reported to a 3.7 client that talks to a 3.6/3.5 server?

Reply:
I will double-check, but from the top of my head: no error, just missing stat data.  (Charles-Henri, who reported ZOOKEEPER-4026, discovered the problem while adding `multi` support to Kazoo.  The "2" in front of the request just fell off :)

Reply:
I have double-checked this, and my recollection was correct: a new client requesting a `Stat` from 3.5, 3.6 or current `master` receives a response, but it is missing `Stat` data.